ci-operator/step-registry/hypershift/hive/workflow/hypershift-hive-workflow-workflow.yaml

```diff
@@ -0,0 +1,21 @@
+workflow:
+  as: hypershift-hive-workflow
+  documentation: |-
+    The HyperShift hive workflow executes tests against a new ephemeral
+    HyperShift cluster. Administrative access to the control plane is provided
+    via the `KUBECONFIG` environment variable.
+
+    Note that tests will execute once the control plane is ready, regardless of
+    whether nodes have been requested, have joined, or are available.
+
+    The control plane is hosted on hive cluster.
+
+    OpenShift versions supported by this workflow are 4.12.x.
```

Comment:
These should be programmatically enforced.  `oc version -o=json` provides the hive cluster's version. `oc adm release info ..` can supply information about the release image specified.  e.g. https://github.com/openshift/release/blob/12b4cdbbfa4ec9bc32b9dd24e080b13b82983cc7/ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-commands.sh#L221-L223 .